src/boot/antelope.ts

```diff
@@ -41,4 +41,11 @@ export default boot(({ app }) => {
     ant.config.setIndexerHealthThresholdSeconds(10);
     ant.config.setIndexerHealthCheckInterval(5000);
 
+    // Finally, we check if the url has the network parameter and if so, we connect to that network
+    // Otherwise we just let the store decide which network to connect to
+    const network = new URLSearchParams(window.location.search).get('network');
+    if (network) {
+        ant.stores.chain.setCurrentChain(network);
+    }
+
```

Comment:
This code allows us to set from outside the network to which the wallet should be connected. This is especially useful when we support multichain.